libs/components/datetime/testing/src/date-range-picker/date-range-picker-harness.ts

```diff
@@ -0,0 +1,194 @@
+import { HarnessPredicate } from '@angular/cdk/testing';
+import { SkyComponentHarness } from '@skyux/core/testing';
+// eslint-disable-next-line @nx/enforce-module-boundaries
+import { SkyDateRangeCalculatorId } from '@skyux/datetime';
+import { SkyInputBoxHarness } from '@skyux/forms/testing';
+
+import { SkyDatepickerHarness } from '../public-api';
+
+import { SkyDateRangePickerFilters } from './date-range-picker-harness.filters';
+
+/**
+ * Harness for interacting with date range picker components in tests.
+ */
+export class SkyDateRangePickerHarness extends SkyComponentHarness {
+  /**
+   * @internal
+   */
+  public static hostSelector = 'sky-date-range-picker';
+
+  #getStartDateInput = this.locatorFor(
```

Comment:
Initially, I thought this was pointing to the `input` element, but it's actually the input box harness. Can we be explicit in the name for these and suffix with `InputBoxHarness`? e.g. `#getStartDateInputBoxHarness`